views/team.erb

```diff
@@ -5,7 +5,14 @@
   <title>Basketball Team</title>
  </head>
  <body>
- 
+ <h1>Team Name: <%=@teamData["name"] %> </h1>
+ <h2>Coach: <%=@teamData["coach"]%>
```

Comment:
```suggestion
 <h2>Coach: <%= @teamData["coach"] %>
```

Reply:
Just went in and made those suggested changes